index.bs

```diff
@@ -3242,6 +3287,43 @@ The sample code for generating and registering a new key follows:
       });
 </pre>
 
+## Registration with Platform Authenticator ## {#sample-registration-with-platform-authenticator}
+
+This is flow for when the [=[RP]=] is only interested in creating a public key credential with 
+[=platform authenticators=].
+
+1. The user visits example.com. At this point, the user likely already logged in. 
+
+1. The [=[RP]=] script runs the code snippet below. 
```

Comment:
ought this scenario have steps similar to, or the same as, [11.1 Registration](https://w3c.github.io/webauthn/#sample-registration) step 3 and 4 after the above step ? (ie, for consistency)